junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java

```diff
@@ -191,6 +204,23 @@ public boolean supportsParameter(ParameterContext parameterContext, ExtensionCon
         }
     }
 
+    /**
+     * @see org.junit.jupiter.engine.extension.TestInfoParameterResolver#supportsParameter
+     * @see org.junit.jupiter.engine.extension.RepetitionExtension#supportsParameter
+     * @see org.junit.jupiter.engine.extension.TestReporterParameterResolver#supportsParameter
+     * @see org.junit.jupiter.engine.extension.TempDirectory#supportsParameter
```

Comment:
Just a cosmetic note - maybe format these as `{@code ...}` instead? They are all pack-private classes which IDE doesn't like although it's obviously not an error.

Reply:
didn't know @see and @code go together this way. nice